server/api/users.js

```diff
@@ -1,19 +1,73 @@
 const router = require('express').Router();
 const {
-  models: { User },
+  models: { User, Order, Order_Product },
 } = require('../db');
 module.exports = router;
 
+// GET /api/users
 router.get('/', async (req, res, next) => {
   try {
     const users = await User.findAll({
       // explicitly select only the id and username fields - even though
       // users' passwords are encrypted, it won't help if we just
       // send everything to anyone who asks!
-      attributes: ['id', 'username'],
+      attributes: ['id', 'username', 'email'],
```

Comment:
Maybe we should grab the userType too and display them. An admin should be able to go through the list and see who is a customer and who is an admin (Might be good to have a display admin filter eventually and that would be necessary)--especially if we want to promote a user to an admin!